tools/mothur/tools/mothur/nmds.xml

```diff
@@ -5,47 +5,68 @@
     </macros>
     <expand macro="requirements" />
 
-    <command interpreter="python"><![CDATA[
-        mothur_wrapper.py 
-        --cmd='nmds'
-        --result='^mothur.\S+\.logfile$:'$logfile,'^\S+\.nmds\.axes$:'$nmds_axes,'^\S+\.nmds\.iters$:'$nmds_iters,'^\S+\.nmds\.stress$:'$stress_nmds
-        --outputdir='$logfile.extra_files_path'
-        --phylip="$dist"
-        #if $axes.__str__ != "None" and len($axes.__str__) > 0:
-            --axes="$axes"
-        #end if
-        #if int($mindim.__str__) > 0:
-            --mindim="$mindim"
-        #end if
-        #if int($maxdim.__str__) >= int($mindim.__str__):
-            --maxdim="$maxdim"
-        #end if
-        #if int($iters.__str__) > 0:
-            --iters="$iters"
-        #end if
-        #if int($maxiters.__str__) > 0:
-            --maxiters="$maxiters"
-        #end if
-        #if float($epsilon.__str__) > 0.0:
-            --epsilon="$epsilon"
-        #end if
+    <command><![CDATA[
+        echo "
+        nmds(
+            phylip="$dist"
+            #if $axes:
+                ,axes="$axes"
+            #end if
+
+            ,mindim="$mindim"
+            
+            #if $maxdim >= $mindim:
+                ,maxdim="$maxdim"
+            #end if
+
+            ,iters="$iters"
+            ,maxiters="$maxiters"
+            
+            ## Mothur can't cope with scientific notation
+            #set epsilon_decimal = "{:.12f}".format(float($epsilon))
+            ,epsilon="$epsilon_decimal"
+        )"
+        | sed 's/ //g'  ## mothur trips over whitespace
+        | mothur;
+
+
+        ## move output files to correct destination
+        prefix="$dist";
+        mv mothur.*.logfile "$logfile";
+        mv \${prefix%.dat}*.axes "$nmds_axes";
+        mv \${prefix%.dat}*.iters "$nmds_iters";
+        mv \${prefix%.dat}*.stress "$nmds_stress";
     ]]></command>
     <inputs>
         <param name="dist" type="data" format="lower.dist,square.dist" label="phylip - Distance Matrix"/>
```

Comment:
could you add mothur prefix here? `format="mothur.lower.dist,mothur.square.dist"`

( I added mothur prefixes everywhere already with a script once, except when multiple formats are allowed)